apps/server/src/models/wallets.ts

```diff
@@ -21,6 +21,7 @@ export enum WalletType {
 export enum OwnerType {
   USER = 'user',
   GROUP = 'group',
+  SAVINGS_GOAL = 'savings-goal',
 }
```

Comment:
You should have also refactored this as you did others by not making use of enums but `as const`